gandalf.js

```diff
@@ -4,129 +4,129 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-This function seems to have grown quite complex. Can we break it down further?